cpp/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ option(ENABLE_HDFS "Enable HDFS" OFF)
 option(ENABLE_ORC "Enable ORC" OFF)
 option(ENABLE_ABFS "Enable ABFS" OFF)
 option(ENABLE_GPU "Enable GPU" OFF)
+option(ENABLE_ICEBERG_WRITE "Enable iceberg write" OFF)
```

Comment:
What happens if a user turns it on right now? Given the Velox PR is not landed

Reply:
The compile will fail